components/org.wso2.carbon.identity.conditional.auth.functions.utils/pom.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  ~ Copyright (c) 2023, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Update all license headers to the new license.

Reply:
Resolved via https://github.com/wso2-extensions/identity-conditional-auth-functions/pull/148/commits/9b2f556bc689b5f024cb8bb6daf372b86d7abf46